tests/fv3_conf/cpld_control_run.IN

```diff
@@ -45,10 +45,13 @@ cp    @[INPUTDATA_ROOT]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
 # WW3 fix/input
 if [[ $CPLWAV == .true. ]]; then
   cp    @[INPUTDATA_ROOT_WW3]/mod_def.points .
-  if [[ $OCNRES == 025 ]]; then
-     cp    @[INPUTDATA_ROOT_WW3]/mod_def.mx@[OCNRES]lite mod_def.ww3
+  if [[ $WAVDOMAIN == mx025 ]]; then
+     cp  @[INPUTDATA_ROOT_WW3]/mod_def.@[WAVDOMAIN]lite mod_def.ww3
```

Comment:
So the quarter degree coupled run is using a different mod_def file name "lite", will we consider use the same lite mod def files for other resolutions?